api/alert_infra_conditions.go

```diff
@@ -0,0 +1,121 @@
+package api
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+)
+
+func (i *InfraClient) queryAlertInfraConditions(policyID int) ([]AlertInfraCondition, error) {

```

Comment:
`i` is probably a bad name for this due to its more idiomatic usage, looping, etc, maybe just `c` or `cli` or something that evokes client?